lib/logstash/filters/jdbc_static.rb

```diff
@@ -175,9 +175,23 @@ def loader_runner
   private
 
   def prepare_data_dir
+    # cleanup existing Derby file left behind in $HOME
+    derby_log = "#{ENV['HOME']}/derby.log"
+    if ::File.exist?(derby_log)
+      begin
+        ::File.delete()
```

Comment:
We need to tell it which file to delete :)

```suggestion
        ::File.delete(derby_log)
```

Reply:
:face_with_head_bandage: